packages/image/tsconfig.json

```diff
@@ -0,0 +1,19 @@
+{
+  // Extends can't use node's resolution rn https://github.com/dividab/tsconfig-paths/issues/209#issuecomment-1177261091
+  "extends": "../../node_modules/@webstudio-is/tsconfig/base.json",
+  "include": ["src", "./src/**/__generated__/*.json"],
```

Comment:
since this is only for image package, I wouldn't do it, because this would be unexpected. It should be either fo all packages/apps or for none I think, then it's consistent, but doing it for all is probably not reasonable idk

Reply:
whatyou will likely see is someone wasting hours because importing a json in one package works and in another one doesn't, until they realize it is disabled in that package ... we need to strive for consistency across packages

so I am very much against doing it in one package

Reply:
now allowed only in 2 packages - Image and react-sdk and in both restricted to generated as no real use cases someone would want to import non generated json. This setup forces all to put new json into generated. Imo current is consistent